docs/overrides/main.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base.html" %}
+
+{% block styles %}
+    {{ super() }}
+    <link rel="stylesheet" href="https://www.eclipse.org/public/css/astro.css">
+    <style>
+      html {
+          font-size: 22px;
+      }
+    </style>
```

Comment:
I would add the following because the footer is veeeery small otherwise.
```
    .footer-min p, .footer-min ul {
            font-size: 1em;
     }
```